js/exporter/exceljs/export.js

```diff
@@ -247,15 +268,26 @@ export const Export = {
                 this.setAlignment(excelCell, wrapText, horizontalAlignment);
             }
 
-            if(isFunction(customizeCell)) {
-                customizeCell(privateOptions._getCustomizeCellOptions(excelCell, cell));
-            }
-
             if(privateOptions._needMergeRange(rowIndex, headerRowCount)) {
-                const mergeRange = this.tryGetMergeRange(rowIndex, cellIndex, mergedCells, dataProvider);
-                if(isDefined(mergeRange)) {
-                    mergeRanges.push(mergeRange);
+                if(this.cellInMergeRanges(mergedCells, rowIndex, cellIndex)) {
+                    const cell = mergedCells[rowIndex][cellIndex];
+                    if(cell.merged === false) {
+                        excelCell.style = cell.masterCell.style;
+                        excelCell.value = cell.masterCell.value;
+                    }
+                } else {
+                    const mergeRange = this.tryGetMergeRange(excelCell, mergedCells, rowIndex, cellIndex, dataProvider, mergeRowFieldValues, mergeColumnFieldValues, privateOptions);
```

Comment:
Here we can write 'mergeRanges.updateRanges(rowIndex, cellIndex, colSpan, rowSpan, mergeRowFieldValues, mergeColumnFieldValues)' - imho there is enough code to try a new class

Reply:
9c50f59